src/pages/StreamListingPage.tsx

```diff
@@ -59,37 +61,35 @@ const mapOrderByToIndexer = (orderBy: OrderBy): IndexerOrderBy => {
 }
 
 const mapOrderDirectionToIndexer = (
-    orderDirection: OrderDirection,
+    orderDirection: ListOrderDirection,
 ): IndexerOrderDirection => {
-    if (orderDirection === OrderDirection.Desc) {
+    if (orderDirection === ListOrderDirection.Desc) {
         return IndexerOrderDirection.Desc
     }
 
     return IndexerOrderDirection.Asc
 }
 
-const mapOrderByToGraph = (orderBy: OrderBy): TheGraphOrderBy => {
+const mapOrderByToGraph = (orderBy: ListOrderBy): Stream_OrderBy => {
     switch (orderBy) {
-        case OrderBy.Id: {
-            return TheGraphOrderBy.Id
+        case ListOrderBy.Id: {
+            return Stream_OrderBy.Id
         }
         default:
-            return TheGraphOrderBy.Id
+            return Stream_OrderBy.Id
     }
```

Comment:
That's a fancy way of saying
```ts
return Stream_OrderBy.Id
```

Reply:
I'm guessing that whoever made this originally wanted to future-proof it

Reply:
Looks like it only got worse. 😂